Library/OcCryptoLib/gost3411-2012-const.h

```diff
@@ -0,0 +1,319 @@
+/*
+ * Copyright (c) 2013, Alexey Degtyarev <alexey@renatasystems.org>.
+ * All rights reserved.
+ *
+ * $Id$
```

Comment:
This comment lacks license. Alexey Degtyarev's implementation is licensed under BSD 2 Clause. Your changes should follow. Please replace it In SPDX format with something like this:

```c
/** @file
  Copyright (c) 2013, Alexey Degtyarev <alexey@renatasystems.org>. All rights reserved.<BR>
  Copyright (c) 2022 Maxim Kuznetsov. All rights reserved.<BR>
  SPDX-License-Identifier: BSD-2-Clause
**/
```